Prototope/Layer.swift

```diff
@@ -253,13 +258,23 @@ public class Layer: Equatable, Hashable {
 	}
 
 	/** The scaling factor of the layer along the x dimension. Defaults to 1. */
-	public var scaleX: Double = 1 {
-		didSet { updateTransform() }
+	public var scaleX: Double {
+        get {
+            return layer.valueForKeyPath("transform.scale.x") as Double
```

Comment:
Thanks for having a look at this!

I feel like that’d be an implementation detail that would easily leak when using this API. Say you set the rotation (by setting it at the `layer.transform` level) and then immediately query the rotation (this would extract it from `layer.presentationLayer.transform`) - you will get the old value.

Besides, it feels like the whole point of this was to abstract away some of the UIKit, CA framework weirdness behind an API that’s simpler, quicker to use and understand for a limited set of use cases. Exposing the CA implementation detail of layer/presentationLayer duality seems to go against this goal (to me). But let me know if I’m wrong :)